lib/mayaUsd/render/mayaToHydra/renderOverride.cpp

```diff
@@ -450,6 +450,17 @@ MStatus MtohRenderOverride::Render(const MHWRender::MDrawContext& drawContext) {
     _taskController->SetSelectionColor(_globals.colorSelectionHighlightColor);
     _taskController->SetEnableSelection(_globals.colorSelectionHighlight);
 
+#if USD_VERSION_NUM >= 2005
+    if (_globals.outlineSelectionWidth != 0.f) {
+        _taskController->SetSelectionOutlineRadius(_globals.outlineSelectionWidth);
+        _taskController->SetSelectionEnableOutline(true);
+    } else
+        _taskController->SetSelectionEnableOutline(false);
+#endif
+#if USD_VERSION_NUM >= 1911
+    _taskController->SetColorizeQuantizationEnabled(_globals.enableColorQuantization);
```

Comment:
So, I actually didn't even know this option existed before... however, looking at the docs for `SetColorizeQuantizationEnabled` in `taskController.h`, I saw:

```
    /// Turns the colorize task color quantization on or off.
    /// XXX: This is a temporary function that will be soon deprecated. Please
    //       avoid calling it.
```

...which makes me a little nervous.  However, if this thing is hurting performance, and there's no other way to disable it, I'm not sure what other option we have...?

Reply:
It's in 19.11, 20.02, and 20.05, so not sure how soon it's going to be deprecated, if when it does, can easily ifdef for only those versions it's available?

I believe in 20.05 it became threaded, so it's less painful, but it's horrific before then.

Reply:
Makes sense to me...

Reply:
Just following up on this, as I'm currently reconciling our internal maya-usd branch. This was actually removed as of this core USD commit in the dev branch:
https://github.com/PixarAnimationStudios/USD/commit/584c27d495047aa9a7231df64eec9e22396e5651

I just opened #481 to restrict calling this function to USD versions between 19.11 and 20.05.